src/Compatibility/Microsoft.DotNet.PackageValidation/Microsoft.DotNet.PackageValidation.csproj

```diff
@@ -34,6 +34,15 @@
     <None Include="**/*.props;**/*.targets" Pack="true">
       <PackagePath>%(RecursiveDir)%(Filename)%(Extension)</PackagePath>
     </None>
+    
+    <!-- 
+      Additionally also copy the props and targets from the build folder into buildMultiTargeting so that
+      NuGet will also have these files automatically imported for consuming projects which multitarget
+      (use TargetFrameworks instead of TargetFramework) 
+    -->
+    <None Include="build/*.props;build/*.targets" Pack="true">
```

Comment:
Instead of copying these, can we just have stubs that import the ones in the build folder?

buildMultiTargeting\Microsoft.DotNet.PackageValidation.props
```xml
<Project>
  <Import Project="..\build\$(MSBuildThisFile)" />
</Project>
```
buildMultiTargeting\Microsoft.DotNet.PackageValidation.targets
```xml
<Project>
  <Import Project="..\build\$(MSBuildThisFile)" />
</Project>
```

Reply:
sure, I can do that too.